conf/projects/toradex-verdin-am62/config.conf

```diff
@@ -12,6 +12,8 @@ YOE_PROFILE = "yoe-glibc-systemd-wayland"
 #IMAGE_BOOT_FILES:append = " ${KERNEL_IMAGETYPE}-initramfs-${MACHINE}.bin;${KERNEL_IMAGETYPE}"
 # Use default Device Tree ( LVDS )
 
+TOOLCHAIN:pn-trusted-firmware-a = "gcc"
```

Comment:
what error do we see with clang ? tf-a is supposed to work with clang upstream so we can report it

Reply:
```
/scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/recipe-sysroot-native/usr/bin/aarch64-yoe-linux/aarch64-yoe-linux-llvm-ar cr /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/lib/libc.a /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/abort.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/assert.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/exit.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/memchr.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/memcmp.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/memcpy.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/memcpy_s.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/memmove.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/memrchr.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/memset.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/printf.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/putchar.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/puts.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/snprintf.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strchr.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strcmp.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strlcat.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strlcpy.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strlen.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strncmp.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strnlen.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strrchr.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strtok.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strtoul.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strtoll.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strtoull.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/strtol.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/libc/setjmp.o
aarch64-yoe-linux-ld -o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bl31.elf -z noexecstack 	 -O1 --gc-sections -z common-page-size=4096  -z max-page-size=4096 --fix-cortex-a53-843419 --fix-cortex-a53-835769 --fatal-warnings  -pie --no-dynamic-linker --sort-section=alignment -Map=/scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bl31.map  --script /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bl31/bl31.ld /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/build_message.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bl31_context_mgmt.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bl31_main.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bl31_traps.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/interrupt_mgmt.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/runtime_svc.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/arm_gicv3_common.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/gic-x00.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/gicdv3_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/gicrv3_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/gicv3_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/gicv3_main.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/errata_report.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/context_mgmt.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/cpu_data_array.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/spe.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/sve.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bakery_lock_normal.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/psci_common.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/psci_main.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/psci_mem_protect.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/psci_off.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/psci_on.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/psci_setup.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/psci_suspend.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/psci_system_off.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/plat_gicv3.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/plat_psci_common.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/sec_proxy.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/ti_sci.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/k3_bl31_setup.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/k3_gicv3.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/k3_psci.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/k3_topology.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/arm_arch_svc_setup.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/opteed_common.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/opteed_main.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/opteed_pm.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/std_svc_setup.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bl_common.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/tf_log.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/multi_console.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/pmuv3.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/plat_bl_common.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/plat_log_common.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/plat_common.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/popcountdi2.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/popcountsi2.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/xlat_tables_arch.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/xlat_tables_context.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/xlat_tables_core.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/xlat_tables_utils.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/k3_console.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bl31_entrypoint.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/crash_reporting.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/ea_delegate.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/runtime_exceptions.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/cpu_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/dsu_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/wa_cve_2017_5715_bpiall.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/wa_cve_2017_5715_mmu.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/context.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/cpu_data.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/spinlock.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/psci_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/runtime_errata.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/platform_mp_stack.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/k3_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/opteed_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/debug.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/cache_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/misc_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/platform_helpers.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/cortex_a53.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/cortex_a72.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/enable_mmu.o /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/16550_console.o -L/scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/lib  -lc 
aarch64-yoe-linux-ld: error: unknown argument '--fix-cortex-a53-835769'
make: *** [Makefile:1509: /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/build/k3/lite/release/bl31/bl31.elf] Error 1
make: Leaving directory '/scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/git'
ERROR: oe_runmake failed
WARNING: /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/temp/run.do_compile.816:174 exit 1 from 'exit 1'
WARNING: Backtrace (BB generated script): 
	#1: bbfatal_log, /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/temp/run.do_compile.816, line 174
	#2: die, /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/temp/run.do_compile.816, line 158
	#3: oe_runmake, /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/temp/run.do_compile.816, line 153
	#4: do_compile, /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/temp/run.do_compile.816, line 147
	#5: main, /scratch4/yoe/yoe-distro-toradex/build/tmp/work/verdin_am62-yoe-linux/trusted-firmware-a/2.9+git/temp/run.do_compile.816, line 187
```